dojos.js

```diff
@@ -2148,7 +2148,7 @@ module.exports = function (options) {
             name: dojo.name,
             email: dojo.email
           },
-          event: {},
+          event: event,
```

Comment:
Please change it to an explicit exposure of name rather than the full object :)
The reason for this is simply that we know which variables are actually exposed into emails rather than digging through the DB structure.